### PR TITLE
Convert Readme to Markdown (fix #9)

### LINKED ISSUE #9: Split the README file
Can you please split the README files into subcategories and paragraphs? It's an eyesore.

### PR DESCRIPTION
**Pull request by Croydon**
Convert the readme to a markdown file.

No more wall of text without orientation points 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Options can be any of the following command-line switches:
 Each animation file can be preceded by any combination of the following command-line switches:
```
Comment (Calinou) [inline README.md]: Indent the command-line switch documentation using four spaces, so that it appears as a code block with a fixed-width font.

**Comment by Calinou** [comment]
See the review on the commit.

Some more things:

- In Markdown, you need to create paragraphs by using two line breaks, else, content will stay on the same line.
- Use code blocks for command examples (like `./upgrade output.iqm input.iqm`)
- Link to the LICENSE file for easier browsing using this Markdown snippet:
```
[LICENSE](/LICENSE)
```

**Comment by Croydon** [comment]
Done.

File: README.md (hunk 1 above)
Comment (Croydon) [reply README.md]: You should comment the line which is actually affected. GitHub will notice then when your review is outdated.

**Comment by Croydon** [comment]
Closing since @lsalzman is obviously not interested.

**Pull request closed without merging**